DATA LICENSES.txt

```diff
@@ -0,0 +1,60 @@
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+NON-IBM-LICENSE FOR De-id data files
+
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+TABLE OF CONTENTS
+
+THE REMAINDER OF THIS NON-IBM-LICENSE FILE CONSISTS OF THE FOLLOWING 
+SECTIONS:
+
+ODBL
+CDLA
+CC-BY-SA-3.0
+
+
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+OPEN DATA COMMONS OPEN DATABASE LICENSE (ODBL)
+
+The Program includes some or all of the following works licensed under 
+Open Data Commons Open Database License (ODbL).
+

```

Comment:
Might want to eliminate this blank link between the first sentence and the license URL.  The other sections don't have it (or add a blank line in the other sections).